Simplenote/SimplenoteAppDelegate.m

```diff
@@ -384,12 +386,18 @@ - (IBAction)signOutAction:(id)sender
     }];
 }
 
+- (IBAction)deleteAccountAction:(id)sender {
+    [SPTracker trackDeleteAccountButttonTapped];
+    [self.accountDeletionController requestAccountDeletionFor:self.simperium.user with:self.window];
+}
+
 -(void)signOut
 {
     [SPTracker trackUserSignedOut];
     
     // Remove WordPress token
-    [SPKeychain deletePasswordForService:SPWPServiceName account:self.simperium.user.email];
+
+    [self removeStoredCredentialsFromKeychain];
```

Comment:
I would also clean up account deletion request when we signOut ourselves and when simperium calls `simperiumDidLogout`.